PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/FunctionAppConfigFormPanelAdvance.java

```diff
@@ -85,8 +84,8 @@ private void createUIComponents() {
         // TODO: place custom component creation code here
         appServiceConfigPanelAdvanced = new AppServiceInfoAdvancedPanel<>(project, () -> FunctionAppConfig.builder().build());
         appServiceConfigPanelAdvanced.setValidRuntime(Runtime.FUNCTION_APP_RUNTIME);
-        final List<PricingTier> validPricing = AzureFunctionMvpModel.getInstance().listFunctionPricingTier();
-        appServiceConfigPanelAdvanced.setValidPricingTier(validPricing, AzureFunctionMvpModel.CONSUMPTION_PRICING_TIER);
+        final List<PricingTier> validPricing = PricingTier.FUNCTION_PRICING;
```

Comment:
I am not quite understand these two lines

Reply:
`FUNCTION_PRICING` is a static list contains valid pricing tier for function app